drv/fpga-devices/src/ecp5_spi.rs

```diff
@@ -96,14 +100,17 @@ impl<S: SpiServer> Ecp5Driver for Ecp5UsingSpi<S> {
         self.user_design_reset_duration
     }
 
+    #[allow(clippy::unit_arg)]
```

Comment:
For all the `allow(clippy::unit_arg)` variances in this file, it's the same +1 LOC to split the return to a separate line, e.g.
```rust
    fn configuration_read(&self, data: &mut [u8]) -> Result<(), Self::Error> {
        self.configuration_port.read(data)?;
        Ok(())
    }
```
We might as well do that instead?